pkg/storage/storagewrappers/boundedconcurrency.go

```diff
@@ -32,17 +32,17 @@ var (
 )
 
 type boundedConcurrencyTupleReader struct {
-	storage.RelationshipTupleReader
+	storage.Reader
```

Comment:
Same comment as in https://github.com/openfga/openfga/pull/1258/files#r1441147756 - overly constrictive type restriction here.